clients/cli/src/main/java/org/apache/gravitino/cli/GravitinoOptions.java

```diff
@@ -66,9 +67,10 @@ public Options options() {
     options.addOption(createArgOption("V", VALUE, "property value"));
     options.addOption(
         createArgOption(
-            "t", PROVIDER, "provider one of hadoop, hive, mysql, postgres, iceberg, kafka"));
+            "z", PROVIDER, "provider one of hadoop, hive, mysql, postgres, iceberg, kafka"));
```

Comment:
this short-name change seems not reflected in the cli.md?

Reply:
Fixed.